src/pages/WorkingApi.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Why do you import React ?
```suggestion
import  { useEffect, useState } from "react";
```